_posts/2020-05-04-tus-js-client-200.md

```diff
@@ -62,8 +62,10 @@ The main behavior of tus-js-client and the vast majority of its API surface stay
   The function that is passed as the `fingerprint` option into the `Upload`-constructor is not callback-based anymore. Instead it must also return a Promise. If you supply a custom fingerprint function it must have a signature such as:
   ```js
   function fingerprint (file, options) {
+    const p = new Promise()
     const result = ....
-    return Promise.resolve(result)
+    p.resolve(result)
+    return p
```

Comment:
No, this code is incorrect. A promise object has no `resolve` property but the promise class has a static `resolve` method.